zenarahealth.io.whitelabel-custom-mail.json

```diff
@@ -0,0 +1,55 @@
+{
+  "providerId": "zenarahealth.io",
+  "providerName": "Zenara Health",
+  "serviceId": "whitelabel-custom-mail",
+  "serviceName": "White Label - Custom Mail",
+  "version": 1,
+  "logoUrl": "https://zenarahealth.io/assets/zenarahealth.svg",
+  "description": "Configures your domain to work with Zenara Health's white label service with custom mail address.",
+  "variableDescription": "This configuration sets whitelabel for custom domain and email address.",
+  "syncPubKeyDomain": "zenarahealth.io",
+  "records": [
+    {
+      "type": "A",
+      "host": "%host1%",
+      "pointsTo": "%record1%",
+      "ttl": 600
+    },
+    {
+      "type": "A",
+      "host": "%host2%",
+      "pointsTo": "%record2%",
+      "ttl": 600
+    },
+    {
+      "type": "CNAME",
+      "host": "%sendgridhost1%",
+      "pointsTo": "%sendgridrecord1%",
+      "ttl": 600
+    },
+    {
+      "type": "CNAME",
+      "host": "%sendgridhost2%",
+      "pointsTo": "%sendgridrecord2%",
+      "ttl": 600
+    },
+    {
+      "type": "CNAME",
+      "host": "%sendgridhost3%",
+      "pointsTo": "%sendgridrecord3%",
+      "ttl": 600
+    },
+    {
+      "type": "A",
+      "host": "%host3%",
+      "pointsTo": "%record3%",
+      "ttl": 600
+    },
+    {
+      "type": "A",
+      "host": "%host4%",
+      "pointsTo": "%record4%",
```

Comment:
These templates add quite many records to customer domain. It would be nice to describe what the records are used for in variable description. Or at least in the git commit message. Ideally the pull request description could show how example.com would look like after it would be applied by this template. Try to answer the question: if I as a customer would apply this template to my zone what would I get?